### PR TITLE
Two new clown ERTs

### PR DESCRIPTION
**Pull request by WhyIsCaeciliusTaken**
## About The Pull Request
Adds two new ERTs: the Code Banana ERT and the HONK Squad.

Code Banana ERTs are just clowns with centcomm access, megaphones and lube flowers, making them even weaker than interns.

The HONK Squad is much stronger. They get a special clown version of the death commando hardsuit, a new clown BoH, golden bikehorn, bananium eshield, taeclowndo shoes, a pie cannon and a self refilling lube sprayer.

## Why It's Good For The Game
Honk.

Code Banana ERTs offer an even weaker ERT to get ghosts back in the game without screwing over antags too much.

HONK Squads are roughly on par with a Code Red ERT, but offer a funny alternative to make the round more goofy.

## Changelog
:cl:
add: Added two new clown ERTs, code banana and the HONK squad
add: Added some clown themed ERT items
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
Squints... Two?

**Comment by WhyIsCaeciliusTaken** [comment]
One of them is just a bunch of clowns with CC access, the other is a proper ERT with proper (clown themed) gear

File: code/datums/ert.dm [hunk 1]
```diff
@@ -79,4 +79,23 @@
...
\ No newline at end of file
+	polldesc = "an elite Nanotrasen enhanced supersoldier"
+
+/datum/ert/clown
+	roles = list(/datum/antagonist/ert/clown)
+	leader_role = /datum/antagonist/ert/clown
+	teamsize = 7
+	opendoors = FALSE
+	rename_team = "The Circus"
+	mission = "Provide vital morale support to the station in this time of crisis."
```
Comment (PowerfulBacon) [inline code/datums/ert.dm]:
```suggestion
	mission = "Provide vital morale support to the station in this time of crisis"
```

File: code/datums/ert.dm [hunk 2]
```diff
@@ -79,4 +79,23 @@
...
+
+/datum/ert/honk
+	roles = list(/datum/antagonist/ert/clown/honk)
+	leader_role = /datum/antagonist/ert/clown/honk
+	teamsize = 5
+	opendoors = TRUE
+	rename_team = "HONK Squad"
+	mission = "HONK them into submission"
+	polldesc = "an elite Nanotrasen tactical pranking squad"
+	code = "HOOOOOOOOOONK"
```
Comment (PowerfulBacon) [inline code/datums/ert.dm]: Code is only needed if you don't have a polldesc, not that it matters

**Review by PowerfulBacon — APPROVED** [review]
Epic

**Pull request merged**